core/src/main/kotlin/com/justai/jaicf/activator/strategy/ActivationByContextPenalty.kt

```diff
@@ -0,0 +1,115 @@
+package com.justai.jaicf.activator.strategy
+
+import com.justai.jaicf.context.BotContext
+import com.justai.jaicf.helpers.logging.WithLogger
+import com.justai.jaicf.model.activation.Activation
+import com.justai.jaicf.model.activation.ActivationStrategy
+import com.justai.jaicf.model.state.StatePath
+
+/**
+ * Implementation of [ActivationStrategy] by analogy with JAICP DSL.
```

Comment:
Why do we need the reference to JAICP here?
As for me, It's a redundant info that pollutes the doc